### ISSUE TITLE
Course Materials are not being tested for accessibility

### ISSUE THREAD

**Issue by sbelsk** [Submitty/Submitty#6794]
**What problem are you trying to solve with Submitty**
The Course Materials page is not created by default with the sample course, so it does not get tested by the html validators for accessibility.

**Describe the way you'd like to solve this problem**
Course Materials should be enabled in the sample course when running "vagrant up" with a bunch of sample data and files uploaded under different settings. The goal is to feature the different functionalities of the page to increase the coverage of the tests.

**Comment by shailpatels** [comment]
The course materials page does get created and is enabled with the default sample course on both the development instance and on CI.

The e2e accessibility baseline also runs against course materials however I believe this only checking for html semantic correctness and is less powerful then something like WAVE

The tests are also running against the default page which does not have files pre uploaded files
